R/loadReg2.R

```diff
@@ -444,14 +463,17 @@ predictSolute.loadReg2 <- function(
       }
     }
   }
-  
   # Add dates if requested
   if(date) {
     if(!is.data.frame(preds)) {
       preds <- data.frame(fit=preds)
     }
     # prepend the date column
-    preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #predLoad returns the dates, don't need to get them from metadata
```

Comment:
i thought i remembered that with certain arguments predLoad just returned a vector...let me see if i can recreate that. maybe i'm misremembering